### DISCUSSION TITLE
a couple features that would help in this application

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1584] [Ideas]
1) a cascading drop down menu 
        the selection from drop down #1 affects the selection availability on dropdown #2
2) ability to trigger in automations write exactly what was saved to an internal table write to an external table. if this is already available some documentation on how to make it work 

3) using data validation for user input fields that do not inhibit the ability to summit that field as blank. 

4) being able to submit a blank date field to an external database. it is always throwing an error when reading the default value 

Thank you guys keep up the good work!

**Comment by joebudi** [comment]
Thanks for the feedback @AndrewJRamay . Some great suggestions here. I will bring these to our next iteration meeting.
